src/index.js

```diff
@@ -1,93 +1,16 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
+import express from 'express';
+import bodyParser from 'body-parser';
 
-const isProduction = process.env.NODE_ENV === "production";
```

Comment:
why did you remove the production configurations? We will need them when setting up a prod environment.

Reply:
Can i put it back up and push it?